### ISSUE TITLE
[velbus] For examples to be fully-featured they should demonstrate use of Thing properties

### ISSUE THREAD

**Issue by nagisa** [openhab/openhab-addons#15896]
See the `TODO` reference to this issue in the velbus module’s README.

In short, there’s an example that reads along the lines of

```
Bridge velbus:bridge:1 [ port="COM1" ] {
    vmb2ble     01
    ...
}
```

however the specific modules can also accept various properties (such as those specifying subaddresses.) The example should ideally demonstrate specification of these properties.

**Comment by andrewfg** [comment]
@nagisa it sound like you could be the ideal person to volunteer to write such examples ;)

**Comment by nagisa** [comment]
Yeah, please assign. Filed this issue entirely upon request and to keep track of the TODO in the source code.

**Comment by MDAR** [comment]
@nagisa 
Purely out of curiosity, is there a solid reason for adding Module Things in the Bridge Thing text file? (as opposed to adding via the Main UI where all these properties can be added / edited with ease.

For example, are you creating a script to automatically create Things files from a VelbusLink config file?

**Comment by nagisa** [comment]
I'm allergic to the mouse. As a person close to code I'm also much more comfortable with textual representation of things. It brings me a peace of mind knowing that I can put these text files in another system, plug the cables and get back an instantly working system (even if the previous one happened to meet with its maker.) In fact my entire system is configured this way, not just OpenHAB, and the move from an older server to the new one took me exactly 10 minutes to get back to a fully configured system (plus the time it took to copy over the media files…)

**Comment by MDAR** [comment]
> I'm allergic to the mouse. As a person close to code I'm also much more comfortable with textual representation of things. It brings me a peace of mind knowing that I can put these text files in another system, plug the cables and get back an instantly working system (even if the previous one happened to meet with its maker.) In fact my entire system is configured this way, not just OpenHAB, and the move from an older server to the new one took me exactly 10 minutes to get back to a fully configured system (plus the time it took to copy over the media files…)

I completely understand that.

I am 90% the same, expect that I add the Module Things via the UI so that they can be re-profiled easily when updates to the binding are rolled out.

What I love is that there is no fixed way to achieve the same end goal.

**Comment by AzazelSensei** [comment]
Opened https://github.com/openhab/openhab-addons/pull/21611 with channel names, refresh, dimspeed, counters, and a glass-panel subaddress in the full example.
